app.js

```diff
@@ -1,75 +1,76 @@
-			var watchID = navigator.geolocation.watchPosition(function(position) {
-				loc.lat = position.coords.latitude;
-				loc.lon = position.coords.longitude;
-
-				console.log("lat: " + loc.lat);
-				console.log("lon: " + loc.lon);
-			});
+    lat: null,
+    lon: null,
+    // get user's location if possible
+    getLocation: function () {
+        if ("geolocation" in navigator) {
+            var watchID = navigator.geolocation.watchPosition(function (position) {
+                loc.lat = position.coords.latitude;
+                loc.lon = position.coords.longitude;
+
+                console.log("lat: " + loc.lat);
+                console.log("lon: " + loc.lon);
+            });
+        }
     }
-  }
 };
 
 
 var restaurants = [{
-  name: "McDonald's",
-  image:"url",
-  foodtype:"American",
-  description:"fast food",
-  link:"url"
+    name: "McDonald's",
+    image: "url",
+    foodtype: "American",
+    description: "fast food",
+    link: "url"
 }];
 
 function getNearbyRestaurants() {
-  return restaurants[Math.floor(Math.random()*restaurants.length)];
+    return restaurants[Math.floor(Math.random() * restaurants.length)];
 }
 
-
-$(document).ready(function(){
-
-
-
-$().ready(function() {
+$().ready(function () {
     loc.getLocation();
-}
-  
+    nearbyLoc();
   
-$("#findfood").click(function() {
-    var resultsBody = $("<div>");
 
-    var resultsName = $("<h2>").text(getNearbyRestaurants().name);
-    resultsName.addClass("");
+$("#findfood").click(function () {
+            var resultsBody = $("<div>");
 
-    var resultsImage = $("<p>").text(getNearbyRestaurants().image);
-    resultsImage.addClass("");
+            var resultsName = $("<h2>").text(getNearbyRestaurants().name);
+            resultsName.addClass("");
 
-    var resultsFood = $("<p>").text(getNearbyRestaurants().foodtype);
-    resultsFood.addClass("");
+            var resultsImage = $("<p>").text(getNearbyRestaurants().image);
+            resultsImage.addClass("");
 
-    var resultsDescription = $("<p>").text(getNearbyRestaurants().description);
-    resultsDescription.addClass("");
+            var resultsFood = $("<p>").text(getNearbyRestaurants().foodtype);
+            resultsFood.addClass("");
 
-    var resultsLink = $("<p>").text(getNearbyRestaurants().link);
-    resultsLink.addClass("");
+            var resultsDescription = $("<p>").text(getNearbyRestaurants().description);
+            resultsDescription.addClass("");
 
-    resultsBody.append(resultsName, resultsImage, resultsFood, resultsDescription, resultsLink);
-    $('#results').append(resultsBody);
- 
-  });
+            var resultsLink = $("<p>").text(getNearbyRestaurants().link);
+            resultsLink.addClass("");
 
+            resultsBody.append(resultsName, resultsImage, resultsFood, resultsDescription, resultsLink);
+            $('#results').append(resultsBody);
 
+        });
 
 });
 
-    nearbyLoc();
-    });
 
-function nearbyLoc(){
+
+function nearbyLoc() {
```

Comment:
Can we remove this now that we have a functional zomato.js file?

Reply:
yes